antenna-selection/antennas.py

```diff
@@ -0,0 +1,44 @@
+# Import networkx for graph tools
+import networkx as nx
+
+# Import dwave_networkx for d-wave functions
+import dwave_networkx as dnx
+
+# Import matplotlib.pyplot to draw graphs on screen
+import matplotlib.pyplot as plt
+
+# Set the solver we're going to use
+from dwave.system.samplers import DWaveSampler
+from dwave.system.composites import EmbeddingComposite
+
+sampler = EmbeddingComposite(DWaveSampler())
+
+# Create empty graph
+G = nx.Graph()
+
+# Add edges to graph - this also adds the nodes
+G.add_edges_from([(1, 2), (1, 3), (2, 3), (3, 4), (3, 5), (4, 5), (4, 6), (5, 6), (6, 7)])
```

Comment:
it would be nice to include this graph in readme

Reply:
Also a few words on what this graph represents physically: why these particular edges?

Reply:
I have added the problem and description into the readme now